app-cdr/daa2iso/daa2iso-0.1.7e-r1.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+DESCRIPTION="Convert CD images from daa/gbi to iso"
+HOMEPAGE="http://aluigi.altervista.org/mytoolz.htm"
+SRC_URI="http://aluigi.altervista.org/mytoolz/${PN}.zip -> ${P}.zip"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux ~x64-macos"
+
+DEPEND="app-arch/unzip"
+
+S="${WORKDIR}"
+
+src_compile() {
+	emake -C src -f "${FILESDIR}/Makefile"
```

Comment:
Yes, there are some buggy packages around here. Note though that in the majority of cases, the `configure` script will figure out the right thing, or you'd be using some fancy build system with eclass that passes the correct CC over.